### PR TITLE
task_14

### PR DESCRIPTION
**Pull request by Omneya-Abdelmajeed**
-make the form of cars create multi lingual  
its to hard to change the image input file into Arabic as it needs CSS and javascript 😞 . 
so I hope that using:
            <label for="files" class="btn btn-info" style="background-color: gray; border: 2px solid gray;"> 
                 {{__('cars.chooseFileButton')}}</label>
            <input id="files" style="visibility:hidden;" type="file" name="image">
is a suitable way.
- optional task I will make it soon (إن شاء الله)

### CODE REVIEW

**Comment by Nora-Lootfy** [comment]
I think..., I make you suffer
فدايااااا
image

**Comment by Omneya-Abdelmajeed** [comment]
@Nora-Lootfy 
Is that ok!
لكي يطمئن قلبي بس 🤣

**Comment by Nora-Lootfy** [comment]
It is okay just for U ya @Omneya-Abdelmajeed 😊

File: resources/views/add_car.blade.php [hunk 1]
```diff
@@ -31,29 +31,29 @@
...
         @csrf
         <div class="form-group mb-3 row">
-            <label for="" class="form-label col-md-2 fw-bold text-md-end">Car Title:</label>
+            <label for="" class="form-label col-md-2 fw-bold text-md-end">{{ __('cars.carTitle')}}</label>
             <div class="col-md-10">
-              <input type="text" placeholder="BMW" class="form-control py-2" name="carTitle" value="{{old('carTitle')}}"/>
+              <input type="text" placeholder="{{ __('cars.placeholderCarTitle')}}" class="form-control py-2" name="carTitle" value="{{old('carTitle')}}"/>
               @error('carTitle')
-                <div class="alert alert-warning">{{$message}}</div>
+                <div class="alert alert-warning">{{__('cars.titleRequired')}}</div>
```
Comment (Nora-Lootfy) [inline resources/views/add_car.blade.php:38]: there is a better approach that can be done here
so what if another error happens for example

if the validation rule of decimal fails, the message will not describe what happen

Comment (Nora-Lootfy) [reply resources/views/add_car.blade.php:38]: you can just modify the file in lang/ar/validation.php

**Review by Nora-Lootfy — APPROVED** [review]
All looking good

File: resources/views/add_car.blade.php (hunk 1 above)
Comment (Omneya-Abdelmajeed) [reply resources/views/add_car.blade.php:38]: Sorry, I didn't see your comment @Nora-Lootfy  but I modified and pushed it within session_15, and i will be blessed if you reviewed it I made commenting to the custom errors that exist in the lang/ar/cars.php and modify the code in lang/ar/validation.php as i added attributes

**Pull request merged**